start-configuration

```diff
@@ -45,6 +45,26 @@ if [[ $PROXY ]]; then
     sleep 5
 fi
 
+if [[ $RCON_PASSWORD_FILE ]]; then
+    log ""
+    if [ ! -e ${RCON_PASSWORD_FILE} ]; then
+	log "Initial RCON password file ${RCON_PASSWORD_FILE} does not seems to exist."
+	log "Please ensure your configuration."
+	log "If you are using Docker Secrets feature, please check this for further information: "
+	log " https://docs.docker.com/engine/swarm/secrets"
+	log "ENV value RCON_PASSWORD would be used."
+	log ""
+    else
+	if [[ ${RCON_PASSWORD} ]]; then
+	    
+	    log "RCON_PASSWORD will be covered by values from password file ${RCON_PASSWORD_FILE}"
```

Comment:
This log may be overly concerning since most users will probably just leave the default `RCON_PASSWORD`. I'd say just remove this conditional block.